app/scripts/modules/core/src/pipeline/config/preconditions/selector/preconditionSelector.directive.js

```diff
@@ -45,13 +46,24 @@ module.exports = angular.module('spinnaker.core.pipeline.config.preconditions.se
       return preconditionConfig ? preconditionConfig.contextTemplateUrl : '';
     };
 
+    this.clusterChanged = function (clusterName) {
+        let clusterFilter = appListExtractorService.monikerClusterNameFilter(clusterName);
+        let moniker = _.first(appListExtractorService.getMonikers([$scope.application], clusterFilter));
```

Comment:
just `let moniker = first(...` - don't need the `_.` since you're explicitly importing it above.

Reply:
yup, overlooked this one.  @anotherchrisberry.  I've fixed it.